src/IssueViz.Security/Commands/IssueVizSecurityPackage.vsct

```diff
@@ -16,6 +16,14 @@
         </Strings>
       </Button>
 
+      <Button guid="guidHotspotsCmdSet" id="TaintToolWindowCommandId" priority="0x0101" type="Button">
```

Comment:
Copy of the existing command with the priority incremented by one so it appears after it in the menu.